compose/opensearch.yml

```diff
@@ -0,0 +1,38 @@
+version: "3"
+services:
+  opensearch-node:
+    image: docker.io/opensearchproject/opensearch:latest
+    hostname: opensearch-node
+    environment:
+      discovery.type: single-node
+      # Reference: https://opensearch.org/docs/latest/security/configuration/demo-configuration/#setting-up-a-custom-admin-password
+      OPENSEARCH_INITIAL_ADMIN_PASSWORD: password4Opense@rch
+      OPENSEARCH_JAVA_OPTS: >
+        -Dcom.sun.management.jmxremote.autodiscovery=true
+        -Dcom.sun.management.jmxremote
+        -Dcom.sun.management.jmxremote.port=9292
+        -Dcom.sun.management.jmxremote.rmi.port=9292
+        -Djava.rmi.server.hostname=opensearch-node
+        -Dcom.sun.management.jmxremote.authenticate=false
+        -Dcom.sun.management.jmxremote.ssl=false
+        -Dcom.sun.management.jmxremote.local.only=false
+    volumes:
+      - opensearch-data:/usr/share/opensearch/data
+    ports:
+      - 9200:9200 # REST API
+      - 9600:9600 # Performance Analyzer
+
+  opensearch-dashboards:
+    image: docker.io/opensearchproject/opensearch-dashboards:latest
+    hostname: opensearch-dashboards
+    ports:
+      - 5601:5601
+    expose:
+      - "5601"
```

Comment:
I think being present in `ports` already implies it's accessible to other containers, so the `expose` becomes redundant?

Reply:
Yea, `port` should be sufficient for forwarding. Does not seem like any containers accessing this port anyway :D